Index: src/db_file.c
===================================================================
--- src/db_file.c
+++ src/db_file.c	2010-04-12 18:23:06.000000000 +0200
@@ -701,7 +701,7 @@
   }
 
   if(tmpstr){
-    retval=dofprintf(tmpstr);
+    retval=dofprintf(tmpstr, "%s");
     free(tmpstr);
     return retval;
   }else {
@@ -740,7 +740,7 @@
 
 
   tmpstr=encode_base64((byte *)ptr,strlen(ptr));
-  retval=dofprintf(tmpstr);
+  retval=dofprintf(tmpstr, "%s");
   free(tmpstr);
   free(ptr);
 
